azurerm/resource_arm_container_registry_test.go

```diff
@@ -221,6 +222,27 @@ func TestAccAzureRMContainerRegistry_update(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMContainerRegistry_geoReplication(t *testing.T) {
+	ri := acctest.RandInt()
+	georeplicationLocations := []string{"\"eastus\"", "\"westus\""}
+	config := testAccAzureRMContainerRegistry_geoReplication(ri, testLocation(), "Premium", strings.Join(georeplicationLocations[:], ","))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMContainerRegistryDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMContainerRegistryExists("azurerm_container_registry.test"),
+					testCheckAzureRMContainerRegistryGeoreplicationExists("azurerm_container_registry.test", georeplicationLocations),
```

Comment:
Consider adding test case for the scenario where the user switches skus and updates to the geo-replication locations

Reply:
Also you may want to consider verifying some values, not just their existence.